Makefile

```diff
@@ -56,4 +63,9 @@ docker-build: build
 docker-push:
 	docker push ${IMAGE}
 
+deploy: docker-build docker-push
+	sed 's#\(image:\) .*#\1 $(IMAGE)#' /tmp/deis-$(SHORT_NAME) | kubectl apply --validate=true -f -
```

Comment:
FYI eventually we can replace this with `kubectl set image rc deis-$(SHORT_NAME)`, if we can't already.